src/ape/api/address.py

```diff
@@ -9,10 +9,25 @@
 
 @abstractdataclass
 class AddressAPI:
+    """
+    A base address API class. All account-types subclass
+    this type, including externally owned accounts as well
+    as contract accounts.
```

Comment:
actually there isn't any other type of account than these two lol :sweat:

Reply:
ha so maybe like `which includes both ...`

Reply:
I don't think this do necessarily suggests there are other potential types

Reply:
Maybe we just don't need the `including` part ?

Reply:
yeah, sounds good